CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+# Contribution Guidelines
+
+These coding conventions are meant to serve as a directive to improve our program quality and our productivity. Prime consideration is given to the issues of program readability, understanding, ease of development and maintenance, ease of debugging, and program efficiency. 
+
+Good programming practices and established conventions and standards should always be followed. Changes to improve machine efficiency should never obscure organized structured program logic.
+
+These standards, conventions, and guidelines will provide for uniformity in the development, construction and installation of reliable program.
+
+## Pull requests
+
+Please ensure the following steps are carefully followed when making a PR.
+
+### Only touch relevant files
+
+Make sure your PR stays focused on a single feature. Don't change project configs or any files unrelated to the subject you're working. Open a single PR for each subject.
+
+### Make sure your code is clean
+
+Checkout the [project style guide](TO DO - insert coding convention guidelines), make sure your code is conformant and clean. Remove any debugging lines (`debuggers`, `console.log`).
+
+### Make sure you unit test your changes
+
+Adding a feature? Make sure you add unit tests to support it.
+
+Fixing a bug? Make sure you added a test reproducing the issue.
+
+### Make sure tests pass
+
+All our projects' unit tests can be run by typing `yarn test` at the root of the project. You may need to install dependencies like `mocha`, `chai`, or `sinon`.
+
+### Keep your commit history short and clean
+
+In a large project, it is important to keep the git history clean and tidy. This helps to identify the causes of bugs and helps in identifying the best fixes.
+
+Keeping the history clean means making one commit per feature. It also means squashing every fix you make on your branch after team review.
+
+### Be descriptive
+
+Write a convincing description of your PR and why we should land it.
+
+### Code review
+
+It is important for us to keep the core code clean and consistent. Please ensure any pull request have been reviewed by at least one of the team members.
+
+### Steps for pull request
+
+Above steps should be used when following the steps belowwhen you'd like your work considered for inclusion in the
+project:
+
+1. [Fork](http://help.github.com/fork-a-repo/) the project, clone your fork,
+   and configure the remotes:
```

Comment:
Do we need people to fork, or do we just clone and use feature branches? I think the assignment brief suggests the latter.

Reply:
NVM, we do need to fork